test/python/transpiler/test_preset_passmanagers.py

```diff
@@ -51,6 +51,23 @@ def test_no_coupling_map(self, level):
         result = transpile(circuit, basis_gates=['u1', 'u2', 'u3', 'cx'], optimization_level=level)
         self.assertIsInstance(result, QuantumCircuit)
 
+    @data(3)
+    def test_layout_3239(self, level):
```

Comment:
```
    @data(3)
    def test_layout_3239(self, level):
```

I think is this equivalent to
```
    def test_layout_3239(self, level=3):
```

Reply:
Yeah, I had put it in that format at first because I saw other tests in the file having a similar format (`@data(0, 1, 2, 3)`), so I wanted to stay consistent with the rest of the tests. But I guess those other tests may have only been that way because they had the test go through all 4 levels. I'll edit the file to just have `def test_layout_3239(self, level=3):`